### DISCUSSION TITLE
Is there a way format html.erb files on save?

### DISCUSSION THREAD

**Discussion by ben-flyht** [zed-industries/zed#6677] [Feature Requests]
Formatting of js files and rb files seems to come out of the box. I've installed Solagraph, but struggling to get html.erb files to automatically format on save. What am I missing?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
